packages/material-ui/src/styles/experimentalStyled.d.ts

```diff
@@ -180,7 +180,9 @@ export interface StyledOptions {
 }
 
 interface MuiStyledOptions<Theme extends object = any> {
-  muiName: string;
+  displayName?: string;
+  className?: string;
+  muiName?: string;
```

Comment:
Could you add tests what each of these are for? Especially why we need 3 fields for that.

Reply:
Will do

Reply:
Added